src/components/ChangePlan/PlanCalculator.styles.js

```diff
@@ -11,3 +11,10 @@ export const PlanTabContainer = styled.section`
     }
   }
 `;
+
+export const HiddenTooltip = styled.div`
+  .disabled {
+      display: none;
+    }
+  }
```

Comment:
You have an extra `}` here. That is why tests are failing 😅

Reply:
Thanks Lu! That was because I merged with kdiff3 and I left an extra } . Thank you very much! 🙏